x/doc.go

```diff
@@ -0,0 +1,5 @@
+package x
+
+/*
+Package x is an experimental package for early-stage features. The API here is not stable and may change.
+*/
```

Comment:
```suggestion
// Package x is an experimental package for early-stage features. The API here is not stable and may change.
package x
```
no actual preference for `//` vs `/*` (`/*` is kinda nice for big blobs of text tho), but it needs to be above the package definition to be a package-doc / to appear in godoc displays.